src/i18n/swg-strings.js

```diff
@@ -89,4 +89,68 @@ export const SWG_I18N_STRINGS = {
     'zh-hk': '透過 Google 提供內容',
     'zh-tw': '透過 Google 捐款',
   },
+  'REGWALL_REGISTERED_BEFORE_LANG_MAP': {
+    'en': 'You have registered before.',
+    'ar': 'لقد سبق أن تسجّلت.',
+    'de': 'Du bist bereits registriert.',
+    'en-au': 'You have registered before.',
+    'en-ca': 'You have registered before.',
+    'en-gb': 'You have registered before.',
+    'en-us': 'You have registered before.',
+    'es': 'Ya te habías registrado anteriormente.',
+    'es-419': 'Ya te registraste antes.',
+    'fr': 'Vous vous êtes déjà inscrit.',
+    'fr-ca': 'Vous vous êtes inscrit auparavant.',
+    'hi': 'आपने पहले ही इसके लिए रजिस्टर कर लिया है.',
+    'id': 'Anda telah mendaftar sebelumnya.',
+    'it': 'Registrazione già effettuata in precedenza.',
+    'jp': 'すでに登録済みです。',
+    'ko': '이전에 등록한 사용자입니다.',
+    'ms': 'Anda telah mendaftar sebelum ini.',
+    'nl': 'Je hebt je al eerder geregistreerd.',
+    'no': 'Du er allerede registrert.',
+    'pl': 'Masz już wcześniejszą rejestrację.',
+    'pt': 'Já se registou anteriormente.',
+    'pt-br': 'Você já tem um cadastro.',
+    'ru': 'Вы уже зарегистрированы.',
+    'se': 'Du har redan registrerat dig.',
+    'th': 'คุณเคยลงทะเบียนแล้ว',
+    'tr': 'Daha önce kaydolmuştunuz.',
+    'uk': 'Ви вже зареєструвалися раніше.',
+    'zh-cn': '您之前已注册。',
+    'zh-hk': '您之前已註冊。',
+    'zh-tw': '你已註冊這個出版品。',
+  },
+  'NEWSLETTER_SIGNED_UP_BEFORE_LANG_MAP': {
```

Comment:
same here, "ALREADY_SIGNED_UP" seems more natural. I'd also consider using "ALREADY_REGISTERED" here as well if applicable, if the terminology fits with the use case.

Reply:
Done